liqd_product/apps/partners/templates/partner_landing_page.html

```diff
@@ -31,6 +31,28 @@ <h2 class="aside__block__title">{% trans 'Activity' %}</h2>
                     {% endfor %}
                 </div>
             </div>
+
+            <div class="aside__block">
+                <h2 class="aside__block__title">Statistiken</h2>
```

Comment:
Did we decide not to translate Strings here or is that just because of the early state?